usr/sbin/truenas-install

```diff
@@ -962,7 +962,7 @@ report_installation()
                     --arg h "${hash}" \
                     --argjson v "[{\"version\": \"$version\"}]" \
                     '{system_hash: $h, platform: $p, "install": $v}' )
-    echo $usage_str | curl -X POST -m 15 -d '@-' http://usage.freenas.org/submit
+    echo $usage_str | curl -X POST -m 15 -d '@-' https://usage.truenas.org/submit
```

Comment:
I think it was supposed to be truenas.com

Reply:
Yup, was making an iso right now to test. Going to do that again :P Thank you for letting me know :)